Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -1039,6 +1039,9 @@ protected class ClassWriter : IClassWriter {
       AddTestCheckerIfNeeded(m.Name, m, wr);
       var typeParameters = TypeParameters(TypeArgumentInstantiation.ToFormals(ForTypeParameters(typeArgs, m, lookasideBody)));
       var parameters = GetMethodParameters(m, typeArgs, lookasideBody, customReceiver, returnType);
+
+      if (!createBody && m is Constructor) { return null; }
```

Comment:
This could also be done one level up, but even though it doesn't break any tests, it's unclear to me if that's valid for all compilers.